mangaki/mangaki/management/commands/compare.py

```diff
@@ -12,13 +12,14 @@
 from mangaki.utils.als import MangakiALS
 from mangaki.utils.knn import MangakiKNN
 from mangaki.utils.svd import MangakiSVD
+from mangaki.utils.pca import MangakiPCA
+from mangaki.utils.zero import MangakiZero
 
 from mangaki.utils.values import rating_values
-from mangaki.utils.zero import MangakiZero
 
 import matplotlib.pyplot as plt
 
-TEST_SIZE = 50000
+TEST_SIZE = 0.2
```

Comment:
A comment on what purpose does it serve or what does the value means? (moving from 5k to 0.2 is strange.)